idr/tests/test_attributes.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+from attributes import get_phenotypes_for_genelist
+from attributes import get_phenotypes_for_gene
+from attributes import get_similar_genes
+from attributes import get_organism_screenids
+from connections import create_http_session, connection
+from images import images_by_phenotype
+
+from config import idr_base_url, organism
+
+
+class TestAttributes():
+
+    @classmethod
+    def setup_class(cls):
+        cls.session = create_http_session(idr_base_url)
+        cls.conn = connection('idr.openmicroscopy.org', 'public', 'public')
```

Comment:
@joshmoore are we officially publishing the public login now?

Reply:
We should discuss it. But in discussions with @bramalingam we *weren't* ready for this. Please force push this away and we can discuss afterwards.

Reply:
Oh this was a mistake! Sorry about that, testing locally and pushing it the same way!